src/components/modalDialog/SampleModalDialog.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { TextField, Button } from '@mui/material';
+import { GenericModalDialog } from './GenericModalDialog';
+
+
+export const SampleModalDialog:React.FC = () => {
+    const [open, setOpen] = React.useState(false);
+  
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+
+    const handleOnCancel = () => {
+      alert('cancelé')
+      setOpen(false);
+    };
+
+    const handleOnClose = () => {
+      alert('acepté')
+      setOpen(false);
+    };
+  
+    return (
+      <div style={{display: "flex", justifyContent: "center"}}>
+        <Button variant="outlined" onClick={handleClickOpen}>
+          Abrir Dialogo Modal
+        </Button>
+        <GenericModalDialog
+                        isOpen={open}
+                        onClose={handleOnClose}
+                        onCancel={handleOnCancel}
+                        title="Ejemplo de Dialogo Modal"
+                        subtitle="Texto que se quiera mostrar en el contenido del Dialogo"
+                        children={ <>
+                                   <TextField
+                                      required
+                                      id="required"
+                                      label="Domicilio"
+                                      defaultValue="Calle"
+                                      variant="standard"
+                                    />
+                                    <TextField
+                                      id="field-2"
+                                      label="Piso"
+                                      defaultValue="Piso"
+                                      variant="standard"
+                                    />
+                                    <TextField
+                                      id="field-3"
+                                      label="Depto"
+                                      defaultValue="Depto"
+                                      variant="standard"
+                                    />
+                                    </>
+                                  }
+                        /> 
```

Comment:
Genial!! Yo lo usaría así:

```suggestion
                        /> 
                                   <TextField
                                      required
                                      id="required"
                                      label="Domicilio"
                                      defaultValue="Calle"
                                      variant="standard"
                                    />
                                    <TextField
                                      id="field-2"
                                      label="Piso"
                                      defaultValue="Piso"
                                      variant="standard"
                                    />
                                    <TextField
                                      id="field-3"
                                      label="Depto"
                                      defaultValue="Depto"
                                      variant="standard"
                                    />
                       </GenericModalDialog> 
```

Reply:
> Genial!! Yo lo usaría así:

En realidad, puse esto a modo de ejemplo, pero se puede mandar directamente cualquier componente completo (ponele, <About>) como unico children... es solo una formalidad.

Reply:
Sí, sí, sólo aprovechaba para proponer una forma de usarlo apenitas menos verbosa.

Reply:
esta perfecto! yo dejé el children para que supieran que tooooodo lo que vaya ahi, es el cuerpo del mensaje... mas de versiones genericas de dialogos modales que alguna vez usé, solo por eso... cuanto menos mejor, pero quiza menos en lo que hacemos o usaremos, pero en otros casos, es como un modulo mas complejo el que se le pasa al children, solo eso... igual como esta opcional, tampoco modifica mucho